packages/react/src/i18n/translations.ts

```diff
@@ -46,6 +46,8 @@ export const Translations = {
         },
         pagination: {
             results: 'results',
+            of: 'of',
```

Comment:
Suggestion de meme, on serait p-t mieux avec des labels plus complete que des mot partiel ici et la, ca donne un meilleur context et plus facile a maintenir a mon avis.

`results: "{{start}}-{{end}} of {{results}} results"`